result/yoda.js

```diff
@@ -0,0 +1,74 @@
+if ("red" === color) {
+    // ...
+}
+if (color === "red") {
```

Comment:
__[development][eslint]__
* Error - Expected literal to be on the left side of ===. (yoda)
* Error - 'color' is not defined. (no-undef)